webpack.mix.js

```diff
@@ -5,7 +5,7 @@ require('laravel-mix-purgecss');
 let purgeCssOptions = {
   enabled: true,
   // List of regex of CSS class to not remove
-  whitelistPatterns: [/^ball-pulse/, /^vs__/],
+  whitelistPatterns: [/^ball-pulse/, /^vs__/, /dot-/],
```

Comment:
`^` means the beginning of the pattern
```suggestion
  whitelistPatterns: [/^ball-pulse/, /^vs__/, /^dot-/],
```